pages/1_13/Key_Mouse_Input.md

```diff
@@ -0,0 +1,120 @@
+# Key & Mouse Input
+
+Forge uses a library called GLFW for its input events. For more in-depth information on the GLFW library, please visit their documentation website: https://www.glfw.org/documentation.html  
+
+With update .146 that forge released two days ago, it is super easy to register key and mouse input within your mod. To keep your files nice and organized I highly suggest that you make a seperate class for registering input. Name it something like InputEventHandler.
```

Comment:
I don't think it's necessary to mention when this was added exactly. 1.13 is still new and new features will be implemented over time. In the end, anyone reading this should be using the most recent version for Forge.

Reply:
Noted, I'll change that